packages/admin/admin/src/table/filterbar/filterBarActiveFilterBadge/FilterBarActiveFilterBadge.tsx

```diff
@@ -1,35 +1,53 @@
-import { ComponentsOverrides, Theme, Typography } from "@mui/material";
-import { WithStyles, withStyles } from "@mui/styles";
+import { ComponentsOverrides, Typography } from "@mui/material";
+import { css, styled, Theme, useThemeProps } from "@mui/material/styles";
+import { ThemedComponentBaseProps } from "helpers/ThemedComponentBaseProps";
 import * as React from "react";
 
-import { FilterBarActiveFilterBadgeClassKey, styles } from "./FilterBarActiveFilterBadge.styles";
+export type FilterBarActiveFilterBadgeClassKey = "hasValueCount";
 
-/**
- * @deprecated Use MUI X Data Grid in combination with `useDataGridRemote` instead.
- */
-export interface FilterBarActiveFilterBadgeProps {
+const HasValueCount = styled("div", {
+    name: "CometAdminFilterBarActiveFilterBadge",
+    slot: "hasValueCount",
+    overridesResolver(_, styles) {
+        return [styles.hasValueCount];
+    },
+})(
+    ({ theme }) => css`
+        display: flex;
+        align-items: center;
+        background-color: ${theme.palette.grey[100]};
+        box-sizing: border-box;
+        text-align: center;
+        border-radius: 4px;
+        padding: 4px 5px;
+        margin-top: -4;
+        margin-bottom: -4;
```

Comment:
I believe this only works in the object-syntax.

```suggestion
        margin-top: -4px;
        margin-bottom: -4px;
```